### PR TITLE
DTLS 1.3: Run 70-test_tls13xxx tests with dtls

### PR DESCRIPTION
**Pull request by fwh-dc**
##### Checklist
- [x] tests are added or updated

### CODE REVIEW

**Comment by fwh-dc** [comment]
@t8m @mattcaswell This one is ready for review.

**Review by t8m — CHANGES_REQUESTED** [review]

File: test/recipes/70-test_tls13hrr.t [hunk 1]
```diff
@@ -41,61 +38,117 @@ use constant {
...
+        $proxy = TLSProxy::Proxy->new(
+            undef,
+            cmdstr(app([ "openssl" ]), display => 1),
+            srctop_file("apps", "server.pem"),
+            (!$ENV{HARNESS_ACTIVE} || $ENV{HARNESS_VERBOSE})
+        );
+    }
+
+    SKIP: {
+        skip "TODO: When ECX is disabled running this test with DTLS will hang"
```
[the remark this answers is not in the document]
Comment (t8m) [reply test/recipes/70-test_tls13hrr.t:84]: @fwh-dc This seems still outstanding. Please do not use plain TODO.

**Review by t8m — APPROVED** [review]

File: test/recipes/70-test_tls13hrr.t [hunk 2]
```diff
@@ -136,15 +189,33 @@ sub hrr_filter
...
-            $hrr_record->sslv2(),
-            $hrr_record->len_real(),
-            $hrr_record->decrypt_len(),
-            $hrr_record->data(),
-            $hrr_record->decrypt_data());
+
+        my $dup_hrr;
+
+        if ($proxy->isdtls()) {
+            printf("FWH:: askldj");
```
Comment (vdukhovni) [inline test/recipes/70-test_tls13hrr.t:196]: Is this `printf` residual debugging?

**Review by vdukhovni — APPROVED** [review]
LGTM, with perhaps a minor nit (debug printf?)

**Review by vdukhovni — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Merged to the feature branch. Thank you for your contribution.

**Pull request closed without merging**